lib/classes/Error.js

```diff
@@ -2,6 +2,40 @@
 const chalk = require('chalk');
 const version = require('./../../package.json').version;
 
+const consoleLog = (message) => {
+  console.log(message); // eslint-disable-line no-console
+};
+
+const writeMessage = (messageType, message) => {
```

Comment:
writeMessage is an unchanged movement of code from the logError method to a private method of the module.  Because it is a private method, it is not externally available.  Given its use by logError, it gets the same exact coverage as before plus overlapping coverage from the new `logWarning` tests.